index.html

```diff
@@ -41,23 +41,24 @@ <h2 class="social"> LET'S CONNECT </h2>
   <section class="containers-works-section">
    <div class="work-container">
 	  <div class="box" id="first-card">
-		 <img src="./image/snapshoot.png" alt="" class="snapshot">
+		 <img src="./image/snapshoot.png" alt="image card" class="snapshot">
 		 <div class="more-details">
 		   <div class="work-description-align-item">
-			   <div class="works-elements">
-				   <h2 class="project-title"> Tonic </h2>
-				   <div class="frame-client">
-					   <span class="client">
-						   CANOPY</span>
-					   <img src="./image/Counter.png" alt="">
-					   <span class="role"> Back End Dev </span>
-					   <img src="./image/Counter.png" alt="">
-					   <span class="year"> 2015 </span>
-				   </div>
-			   </div>
+		     <div class="works-elements">
+			   <h1 class="project-title"> Tonic </h1>
```

Comment:
- [optional] I'm so sorry for raising this, kindly ensure to use header tags `hx` other that` h1` for all your cards. Google crawlers pay a good deal of attention to the `h1` tag because, being the main title of the page, it conveys (or should convey) the essence of what that page is about. A well-composed h1 tag helps Google match your content with relevant searches. ℹ️